src/utils/theme.ts

```diff
@@ -162,27 +162,27 @@ export const darkTheme: Theme = {
   ...defaultTheme,
   color: {
     background: {
-      main: '#221f33',
+      main: 'linear-gradient(169deg, #3e3869 5%, #241938 98%)',
       topMenu: '#443d66',
-      topMenuButton: '#ff884d',
-      card: '#2b2640',
-      button: '#ff884d',
-      closeButton: '#ff884d',
-      selectInputToggleButton: '#ff884d',
-      selectInput: '#443d66',
-      selectInputExpanded: '#1a1726',
-      selectInputImagePlaceholder: '#443d66',
+      topMenuButton: '#ffffff',
+      card: '#262240',
+      button: 'linear-gradient(to bottom, #fdb754, #f18214)',
+      closeButton: '#ffffff',
+      selectInputToggleButton: '#998ae6',
+      selectInput: '#474078',
+      selectInputExpanded: '#130c1d',
+      selectInputImagePlaceholder: '#4f367a',
       textInput: '#1a1726',
       switchInput: '#1a1726',
-      switchInputActiveTab: '#443d66',
+      switchInputActiveTab:
+        'linear-gradient(to bottom, #734fb3, #422d66), linear-gradient(to bottom, #3d265c, #222130)',
       switchInputInactiveTab: 'transparent',
       pill: '#2b2640',
       checkboxInputInactive: '#665c99',
       dropdownHoverColor: '#443d66',
       selectInputExpandedHover: '#443d66',
-      walletButton: 'linear-gradient(to bottom, #6154a3, #292246)',
-      walletChainDropdown: '#1a1726',
-      walletChainButtonActive: '#443d66',
```

Comment:
Any reason we're removing the dark theme versions of the wallet styles?

Reply:
Hmm, good catch here, I'll see this, probably it got removed while migrating the styles from the buidler landing page.